### ISSUE TITLE
Use gradle for the c++ part as well

### ISSUE THREAD

**Issue by coolya** [kbirken/warp#5]
Gradle recently [released](https://blog.gradle.org/introducing-the-new-cpp-plugins) a c++ plugin. 

We might want to simply use that to get rid of the cmake dependency to simplify the build process.

**Comment by krishnanm86** [comment]
Agreed. I can look at it next week and try to get it working. Might also remove all the dependency on the system path
